### PR TITLE
Add remark-lint

### PR DESCRIPTION
**Pull request by dsifford**
Adds a _really_ strict linter for markdown which should keep everything uniform. Let me know what you think.

### CODE REVIEW

**Comment by dsifford** [comment]
@jvoros -- See [this build](https://travis-ci.org/aliemteam/aliemcards/builds/216518996) for an idea on how the errors are logged.

I'm going to work on fixing these errors now.

File: cards/abg-interpretation/card.md [hunk 1]
```diff
@@ -21,7 +20,7 @@ categories:
 
 **Calculate the excess anion gap** = total anion gap - 12 + measured bicarb
 
--   If sum is &gt; 30, then metabolic alkalosis
+-   If sum is > 30, then metabolic alkalosis
```
Comment (dsifford) [inline cards/abg-interpretation/card.md]: Beautifier must have changed this back. Will fix later.

**Comment by jvoros** [comment]
How did you fix so many linting errors so quickly???

**Comment by jvoros** [comment]
This is a great addition to the work flow

I like how the build fails. Keep Travis catching errors when they're made.

**Comment by dsifford** [comment]
@jvoros 

> How did you fix so many linting errors so quickly???

Remark has a package for vscode which autofixes to "sensible defaults". So all I had to do was go through each file, autofix, then make a few changes here and there for the non-default rules.

**Pull request merged**